allennlp/modules/softmax.py

```diff
@@ -0,0 +1,51 @@
+"""
+The Softmax from calypso
+"""
+
+import numpy as np
+import torch
+
+class Softmax(torch.nn.Module):
+    def __init__(self,
```

Comment:
This class could use a docstring describing what it's doing, and when you might want to use this instead of `torch.nn.functional.softmax`.